meta-lmp-base/recipes-support/docker-image-preload/docker-image-preload/docker-image-preload.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Docker container images preloading
+
+After=local-fs.target
+Before=docker.service
+
+ConditionPathExists=/var/sota/import/installed_versions
+ConditionPathExistsGlob=/var/sota/import/*.tar
+
+[Service]
+Type=oneshot
+
+# pre-load container images into the docker data root directory (only works for overlay2 graph driver)
+ExecStart=/bin/sh -c 'mkdir -p /var/lib/docker && for f in `find /var/sota/import -name "*.tar"`; do echo "Compose App: preloading $f" && tar -xf $f -C /var/lib/docker/ && rm $f; done'
```

Comment:
You can move 'mkdir -p /var/lib/docker' to ExecStartPre.

Reply:
+